spark/dl/pom.xml

```diff
@@ -34,7 +34,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.0.0</version>
+            <version>3.2.0</version>
```

Comment:
Nothing special, just upgrade to recommended version. As discussed offline , I reverted it back to 3.0.0